rfd/0018/README.md

```diff
@@ -76,7 +77,11 @@ What we propose here is to make a reserved namespace for labels that have
 special meanings for sdc-docker. These would look like:
 
 ```
-com.joyent.<key>=<value>
+com.joyent.<key>=<value>     # Joyent/Triton-specific use cases
+```
+or
+```
+triton.<key>=<value>         # Triton-specific use cases
```

Comment:
It's not a high priority, but it would be great to transition packages to use the `triton.package` namespace.